From dddf951ea2bac5af988bfd2b771b1b4ea2430c1d Mon Sep 17 00:00:00 2001
From: "Hebbar, Gururaja" <gururaja.hebbar@ti.com>
Date: Thu, 24 May 2012 14:53:05 +0530
Subject: [PATCH 0976/1092] net: cpsw: drop workaround for TLK110 phy rev 1.0

Rev 1.0 of TLK110 PHY was an early version PHY which
has now been replaced with rev 1.1 and above.

Drop special handling for this PHY.

Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
---
 drivers/net/ethernet/ti/Kconfig | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/net/ethernet/ti/Kconfig b/drivers/net/ethernet/ti/Kconfig
index 2594c94..b42252c 100644
--- a/drivers/net/ethernet/ti/Kconfig
+++ b/drivers/net/ethernet/ti/Kconfig
@@ -60,16 +60,6 @@ config TI_CPSW
 	  To compile this driver as a module, choose M here: the module
 	  will be called cpsw.
 
-config TLK110_WORKAROUND
-       tristate "TI TLK110 v1.0 PHY Workaround"
-       depends on TI_CPSW
-       help
-         This supports workaround for TLK110 rev 1.0 PHY.
-
-         This should be used  when TLK110 rev 1.0 PHY is used. In case of
-         higher version of TLK110 PHY, this workaround is not required
-         and disable it.
-
 config TLAN
 	tristate "TI ThunderLAN support"
 	depends on (PCI || EISA)
-- 
1.7.11.2

